app/services/draw-service.js

```diff
@@ -0,0 +1,62 @@
+import Service from '@ember/service';
+import {
+  A
+} from '@ember/array';
+
+/**
+ * Service to aid drawing on map.
+ */
+export default Service.extend({
+
+  init() {
+    this._super(...arguments);
+  },
+
+  /* Construct draw objects for draw-object component */
+  buildDrawObjects: function (newLayers) {
+    let builtObjects = A();
+
+    // Fetch each object
+    Object.keys(newLayers).forEach(key => {
+      let layer = newLayers[key];
+      if (layer._latlngs) {
+        if (layer._latlngs.length > 1) {
+          return builtObjects.pushObject({
```

Comment:
were the return statements in each of these conditions intentional for this forEach loop?

Reply:
good pickup, the final return statement should suffice